src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -684,25 +683,6 @@ function MoneyRequestConfirmationList({
                 interactive={!isReadOnly}
                 numberOfLinesTitle={2}
             />
-=======
-            {!isDistanceRequest &&
```

Comment:
FYI from my testing, this was not needed -> Also, this change comes right next to a LOT of code that was updated in https://github.com/Expensify/App/pull/38543, which is not in staging yet - so I don't believe this is needed in staging YET. I made sure the other changes did fix the issue https://github.com/Expensify/App/issues/40549